commands/releases/info.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+let cli  = require('heroku-cli-util');
+let co   = require('co');
+
+function* run (context, heroku) {
+  // TODO: find out how to get config vars and addons data in apiv3 or deprecate this command
+  let id = (context.args.release || 'current').toLowerCase();
+  id = id.startsWith('v') ? id.slice(1) : id;
+  let release;
+  if (id === 'current') {
+    release = (yield heroku.request({
+      path: `/apps/${context.app}/releases`,
+      partial: true,
+      headers: { 'Range': `version ..; max=1, order=desc` },
+    }))[0];
+  } else {
+    release = yield heroku.request({
+      path:    `/apps/${context.app}/releases/${id}`,
+      headers: {Accept: 'application/json'},
```

Comment:
yeah you're right, this causes a bug where config vars don't display. And I can't use v2 since I can't pass the range header in.